tests/integration/targets/callback_diy/aliases

```diff
@@ -1,2 +1,3 @@
 shippable/posix/group3
 needs/target/callback
+context/controller
```

Comment:
Plugin tests shouldn't need this alias since `ansible-test` auto-detects them as controller tests.